.vimrc

```diff
@@ -68,14 +115,14 @@
 " }
 
 " Use before config if available {
-    if filereadable(expand("~/.vimrc.before"))
-        source ~/.vimrc.before
+    if filereadable(g:spf13_vimrc_before)
+        exec 'source ' . g:spf13_vimrc_before
```

Comment:
err it only seems to work with registers, not variables. I'm not sure exactly why though.